### PR TITLE
Update node config parameters

### PR DESCRIPTION
**Pull request by bowang-666**
Update agentbaker to support new node config parameters, also removed two incorrect parameters.

### CODE REVIEW

File: pkg/agent/datamodel/types.go [hunk 1]
```diff
@@ -544,6 +544,9 @@ type CustomKubeletConfig struct {
 	TopologyManagerPolicy string    `json:"topologyManagerPolicy,omitempty"`
 	AllowedUnsafeSysctls  *[]string `json:"allowedUnsafeSysctls,omitempty"`
 	FailSwapOn            *bool     `json:"failSwapOn,omitempty"`
+	ContainerLogMaxSizeMB *int32    `json:"containerLogMaxSizeMB,omitempty"`
+	ContainerLogMaxFiles  *int32    `json:"containerLogMaxFiles,omitempty"`
+	PodMaxPids            *int32    `json:"podMaxPids,omitempty"`
```
Comment (alexeldeib) [inline pkg/agent/datamodel/types.go:549]: nit -- might be more clear to rename the same field on the other struct, PodPidsLimit

**Review by alexeldeib — APPROVED** [review]
lgtm

**Review by adilliadil — APPROVED** [review]
lgtm

File: pkg/agent/datamodel/types.go (hunk 1 above)
Comment (bowang-666) [reply pkg/agent/datamodel/types.go:549]: Yes it should be PodPidsLimit but it is hard to update API, I'll leave as it is for now.

**Pull request merged**